test/VisualBasic/CodeCracker.Test/Refactoring/ChangeAnyToAllTests.vb

```diff
@@ -0,0 +1,172 @@
+        '    Dim ints = {1, 2}
+        '    Dim query = ints.Any(Function(i) i >= 1)", "
+        '    Dim ints = {1, 2}
+        '    Dim query = Not ints.All(Function(i) i < 1)")>
+        '<InlineData("
+        '    Dim ints = {1, 2}
+        '    Dim query = ints.Any(Function(i) i < 1)", "
+        '    Dim ints = {1, 2}
+        '    Dim query = Not ints.All(Function(i) i >= 1)")>
+        '<InlineData("
+        '    Dim ints = {1, 2}
+        '    Dim query = ints.Any(Function(i) i <> 1)", "
+        '    Dim ints = {1, 2}
+        '    Dim query = Not ints.All(Function(i) i = 1)")>
+        '<InlineData("
+        '    Dim ints = {1, 2}
+        '    Dim query = ints.Any(Function(i) Not True)", "
+        '    Dim ints = {1, 2}
+        '    Dim query = Not ints.All(Function(i) True)")>
+        '<InlineData("
+        '    Dim ints = {1, 2}
+        '    Dim query = ints.Any(Function(i) i > Not(i = 1))", "
+        '    Dim ints = {1, 2}
+        '    Dim query = Not ints.All(Function(i) i <= Not (i = 1))")>
+        '<InlineData("
+        '    Dim ints = {1, 2}
+        '    Dim query = ints.Any(Function(i) True)", "
+        '    Dim ints = {1, 2}
+        '    Dim query = Not ints.All(Function(i) False)")>
+        '<InlineData("
+        '    Dim ints = {1, 2}
+        '    Dim query = ints.Any(Function(i) False)", "
+        '    Dim ints = {1, 2}
+        '    Dim query = Not ints.All(Function(i) True)")>
+        '<InlineData("
+        '    Dim ints = {1, 2}
+        '    Dim query = ints.Any(Function(i) True = (i = 1))", "
+        '    Dim ints = {1, 2}
+        '    Dim query = Not ints.All(Function(i) False = (i = 1))")>
+        '<InlineData("
+        '    Dim ints = {1, 2}
+        '    Dim query = ints.Any(Function(i) (i = 1) = False)", "
+        '    Dim ints = {1, 2}
+        '    Dim query = Not ints.All(Function(i) i = 1)")>
+        '<InlineData("
+        '    Dim ints = {1, 2}
+        '    Dim query = ints.Any(Function(i) False = (i = 1))", "
+        '    Dim ints = {1, 2}
+        '    Dim query = Not ints.All(Function(i) i = 1)")>
+        '<InlineData("
+        '    Dim ints = {1, 2}
+        '    Dim query = ints.Any(Function(i) (i = 1) <> True)", "
+        '    Dim ints = {1, 2}
+        '    Dim query = Not ints.All(Function(i) i = 1)")>
+        '<InlineData("
+        '    Dim ints = {1, 2}
+        '    Dim query = ints.Any(Function(i) True <> (i = 1))", "
+        '    Dim ints = {1, 2}
+        '    Dim query = Not ints.All(Function(i) i = 1)")>
+        '<InlineData("
+        '    Dim ints = {1, 2}
+        '    Dim query = ints.Any(Function(i) (i = 1) <> False)", "
+        '    Dim ints = {1, 2}
+        '    Dim query = Not ints.All(Function(i) (i = 1) = False)")>
+        '<InlineData("
+        '    Dim ints = {1, 2}
+        '    Dim query = ints.Any(Function(i) False <> (i = 1))", "
+        '    Dim ints = {1, 2}
+        '    Dim query = Not ints.All(Function(i) False = (i = 1))")>
```

Comment:
Sorry. Forgot to uncomment before checking in. Revising now. All tests still pass.